waysip/Cargo.toml

```diff
@@ -12,4 +12,4 @@ authors.workspace = true
 
 [dependencies]
 clap = { version = "4.4.10", features = ["derive"] }
-libwaysip.workspace = true
+libwaysip = { path = "../libwaysip" }
```

Comment:
Do not change back to path way, this will make only waysip can be published to waycrate